test/foundry-tests/Hooks.t.sol

```diff
@@ -227,7 +230,7 @@ contract HooksTest is Test, Deployers, GasSnapshot {
     }
 
     function testValidateHookAddressBeforeAndAfterInitialize(uint152 addr) public {
```

Comment:
It was a little unlcear why we have to now add the mask `0x007ffffFfffffffffFffffFFfFFFFFFffFFfFFff`to the address.. We don't need it before bc we require inputs to be uint152 which works nicely with / it guarantees 8 cleared upper bits. Maybe instead we now allow it to be uint160 and mask it or shift it right 9 bits.

Reply:
or add comment?